### DISCUSSION TITLE
upload resolution setting suggestions  25Mbps

### DISCUSSION THREAD

**Discussion by flipper2222222** [GerryDazoo/SlingerPlayer#36] [Q&A]
hi all,
I have a 25Mbs upload on my connection, can anyone suggest the highest possible resolution settings in the config file without going to high to buffer?
Slinger player with slingbox 500 
thanks

**Comment by GerryDazoo** [comment]
1080I

**Reply by flipper2222222** [reply]
I was more thinking someone might have suggestions for these settings to maximise the usage of my 25Mbps upload bandwidth 

Resolution=12
FrameRate=30
VideoBandwidth=2248
VideoSmoothness=59
IframeRate=4

any suggestions?
